registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -400,6 +402,7 @@
     "sha256:2a19654e54a1338ecb7bc5559a7130d7ad55f8a16b6035f4ac6f182bf6f990de": ["v1.1.4"]
     "sha256:c1ef711b16fe0bd082f00baacf23027c85c9c80c8390ca9ea81d57ce793bdf69": ["v1.1.5"]
     "sha256:699e47bc5f10c8ebad7132a96e26624f564328766d31b0bbdc154c8cbb2ff085": ["v1.1.6"]
+    "sha256:c3a0f068d185dfa958141b87d462f023890d11aa3bf5080aec33727ba3d3d7c0": ["v1.10.0"]
```

Comment:
`c3a0f068d185` matches the SHA for kubeadm-bootstrap-controller v1.10.0 in the [staging repository](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:c3a0f068d185dfa958141b87d462f023890d11aa3bf5080aec33727ba3d3d7c0/details) 👍🏻